lightning/src/ln/onion_utils.rs

```diff
@@ -199,6 +231,8 @@ pub(super) fn route_size_insane(payloads: &Vec<msgs::OnionHopData>) -> bool {
 }
 
 /// panics if route_size_insane(paylods)
+// NOTE: I believe since onion messages can be variable size, we'll have to remove the panic
+// mentioned above^
```

Comment:
`route_size_insane` handles that just fine, you just have to check it at the callsite before calling this.